assets/src/scripts/lib/d3-redux.js

```diff
@@ -13,7 +16,7 @@ const subscribe = function (store, callback, args) {
         }
     }
 
-    store.subscribe(handleUpdate);
+    store.subscribe(throttle(handleUpdate));
```

Comment:
I'm sure this would help on WDFN too. Maybe we should prioritize packaging up this?

Reply:
We should see in testing this if there are any issues, but yes I think it'd be worth it add there too.